src/Faker/Provider/bs_BA/Person.php

```diff
@@ -0,0 +1,97 @@
+        "Salih", "Salim", "Samed", "Samir", "Sanjin", "Saša", "Sead", "Šefik", "Selim", "Semir", "Senad", "Sinan", "Siniša",
+        "Slaven", "Slaviša", "Slobodan", "Stjepan", "Stojan", "Suad", "Sulejman", "Tarik", "Tihomir", "Timur", "Tomislav",
+        "Tomo", "Vahid", "Valentin", "Vanja", "Vedad", "Vladimir", "Zdravko", "Zijad", "Zijah", "Zlatan", "Zlatko", "Zoran",
+        "Zuhdija", "Zulfikar", "Zulfo"
+    );
+
+    /**
+     * @link http://www.just-think-it.com/sbn/bosnianf.htm
+     */
+    protected static $firstNameFemale = array(
+        "Abida", "Adela", "Adele", "Ademira", "Adila", "Adina", "Adisa", "Admira", "Adna", "Adrijana", "Advija", "Aida",
+        "Aiša", "Ajla", "Ajna", "Aldijana", "Aldina", "Aleksandra", "Alisa", "Alma", "Almasa", "Almedina", "Almira",
+        "Amela", "Amila", "Amina", "Amira", "Amna", "Amra", "Ana", "Andjelka", "Andrea", "Anesa", "Anita", "Antonija",
+        "Arijana", "Arnela", "Asima", "Asja", "Azema", "Azemina", "Azra", "Badema", "Bahira", "Belma", "Belmina", "Berina",
+        "Besima", "Biljana", "Branka", "Dajana", "Dalila", "Danica", "Dejana", "Dijana", "Dina", "Dinka", "Dragana",
+        "Dragica", "Džana", "Dženana", "Dženeta", "Dženita", "Edina", "Edita", "Ehlimana", "Ela", "Eldina", "Elvira", "Ema",
+        "Emela", "Emina", "Emira", "Emma", "Enesa", "Enida", "Enisa", "Erna", "Esma", "Fadila", "Fahira", "Fahreta", "Farah",
+        "Fatima", "Fehima", "Ferida", "Gorana", "Hadžera", "Halima", "Hana", "Hanifa", "Hatidža", "Hava", "Ifeta", "Ilda",
+        "Ilhana", "Ilma", "Iman", "Indira", "Ines", "Iris", "Irma", "Ismeta", "Iva", "Ivana", "Ivanka", "Ivona", "Izeta",
+        "Jadranka", "Jana", "Jasenka", "Jasmina", "Jasna", "Jovanka", "Kanita", "Katarina", "Lamija", "Lana", "Lara",
+        "Larisa", "Latifa", "Lejla", "Lina", "Ljiljana", "Lorena", "Lutvija", "Maida", "Maja", "Majda", "Marija", "Marijana",
+        "Marisa", "Matea", "Medina", "Medžida", "Meliha", "Melisa", "Merima", "Merjem", "Mersiha", "Mia", "Mila", "Minela",
+        "Mirjana", "Mirna", "Mirnesa", "Mirsada", "Mubina", "Naida", "Natasa", "Nejra", "Nevena", "Nevzeta", "Nezira",
+        "Nihada", "Nikolina", "Nina", "Nives", "Nudžejma", "Petra", "Rahima", "Ranka", "Rejhana", "Rifeta", "Šejla", "Šemsa",
+        "Sabina", "Sadika", "Sajra", "Saliha", "Samija", "Samira", "Samra", "Sandra", "Sanela", "Sanija", "Sara", "Sarah",
+        "Selma", "Selmira", "Semija", "Semira", "Senada", "Senija", "Senka", "Sinolicka", "Sladjana", "Sonja", "Spomenka",
+        "Stojanka", "Suada", "Sumeja", "Suvada", "Taiba", "Tajra", "Tamara", "Tanja", "Tatjana", "Tihana", "Tijana", "Una",
+        "Valentina", "Vanja", "Vasvija", "Vesna", "Vildana", "Zana", "Željka", "Zana", "Zara", "Zdenka", "Zejneba",
+        "Zerina", "Zijada", "Zlata", "Zlatka", "Zorana", "Zorica", "Zuhra", "Zulejha", "Zumreta"
+     );
+
+    /**
+     * @link http://surnames.behindthename.com/submit/names/usage/bosnian
```

Comment:
I already filtered the list to reflect real Bosnian names. The list of names is provided by Federal Bureau of Statistics and it's in public domain. I'll update the source accordingly.